tests/layer_tests/ovc_python_api_tests/test_tf.py

```diff
@@ -1205,3 +1205,103 @@ def test_tf1_output_with_identity(self):
+            tensors = output.get_names()
+
+            assert len(tensors) == 1
+            out_tensor = list(tensors)[0]
+            assert out_tensor == ref_inputs[idx]
+
+        ov_model = convert_model(path, input=["Input:0", "Input_1:0"])
+        for idx, output in enumerate(ov_model.inputs):
+            tensors = output.get_names()
+
+            assert len(tensors) == 1
+            out_tensor = list(tensors)[0]
+            assert out_tensor == ref_inputs[idx]
+
+        ref_inputs = ["Input", "Input_1"]
+        ov_model = convert_model(path, input=["Input", "Input_1"])
+        for idx, output in enumerate(ov_model.inputs):
+            tensors = output.get_names()
+
+            assert len(tensors) == 1
+            out_tensor = list(tensors)[0]
+            assert out_tensor == ref_inputs[idx]
+
+
+    @pytest.mark.nightly
+    @pytest.mark.precommit
+    def test_tf1_from_memory_single_input_name(self):
+        tf.keras.backend.clear_session()
+        tf.compat.v1.reset_default_graph()
+        from openvino import convert_model
+
+        model, _, _ = create_tf_graph_def(None)
+
+        ov_model = convert_model(model)
+
+        ref_inputs = ["Input:0", "Input_1:0"]
+        for idx, output in enumerate(ov_model.inputs):
+            tensors = output.get_names()
+
+            assert len(tensors) == 1
+            out_tensor = list(tensors)[0]
+            assert out_tensor == ref_inputs[idx]
+
+        ov_model = convert_model(model, input=["Input:0", "Input_1:0"])
+        for idx, output in enumerate(ov_model.inputs):
+            tensors = output.get_names()
+
+            assert len(tensors) == 1
+            out_tensor = list(tensors)[0]
+            assert out_tensor == ref_inputs[idx]
+
+        ref_inputs = ["Input", "Input_1"]
+        ov_model = convert_model(model, input=["Input", "Input_1"])
+        for idx, output in enumerate(ov_model.inputs):
+            tensors = output.get_names()
+
+            assert len(tensors) == 1
+            out_tensor = list(tensors)[0]
+            assert out_tensor == ref_inputs[idx]
+
+    @pytest.mark.nightly
+    @pytest.mark.precommit
+    def test_tf1_input_with_identity(self):
+        tf.keras.backend.clear_session()
+        tf.compat.v1.reset_default_graph()
+        from openvino.tools.ovc import convert_model
+
+        with tf.compat.v1.Session() as sess:
+            x = tf.compat.v1.placeholder(tf.float32, [2], 'x')
+            y = tf.compat.v1.placeholder(tf.float32, [2], 'y')
+            input1 = tf.identity(x, name="x_identity")
+            input2 = tf.identity(y, name="y_identity")
+            add = tf.add(input1, input2, name="add")
+
+            tf.compat.v1.global_variables_initializer()
+            model = sess.graph_def
+
+        ov_model = convert_model(model)
+
+        assert ov_model.inputs[0].get_names() == {"x_identity:0", "x:0"}
+        assert ov_model.inputs[1].get_names() == {"y_identity:0", "y:0"}
```

Comment:
I think here we should have the single tensor name for inputs. We should select a tensor name that is close to `Parameter`, i.e. "x:0" and "y:0". Let us modify translator for identity nodes and check if producer for it is `Parameter`, if yes - do not set tensor name.